pkgs/data/themes/catppuccin-gtk/default.nix

```diff
@@ -6,22 +6,22 @@
 , gtk-engine-murrine
 , sassc
 , which
-, tweaks ? [ ]         # can be "nord" "black" "rimless". cannot mix "nord" and "black"
+, tweaks ? [ ]         # can be "nord" "dracula" "black" "rimless" "normal". cannot mix "nord" and "dracula"
```

Comment:
Given that I am here...

These comments look superfluous, because the assertions in the code below explains them.

Reply:
They may be too cryptic to understand for newbies or other people who don't use Nix that heavily.

Alternatively, I could leave a link towards https://github.com/catppuccin/gtk/blob/main/install.sh#L49

Reply:
> They may be too cryptic to understand for newbies or other people who don't use Nix that heavily.

Though if they try to use it incorrectly, it will throw an error.

Reply:
Alright, I'll leave the link only.

Reply:
On second thought, you're right. The comments are useless. The code is clear enough.